Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -9,136 +9,112 @@ import UIKit
 
 class FacilitiesDropdownCell: UICollectionViewCell {
 
-    private enum FacilityName: String {
-        case bowlingLanes = "BOWLING LANES"
-        case equipment = "EQUIPMENT"
-        case fitnessCenter = "FITNESS CENTER"
-        case gymnasium = "GYMNASIUM"
-        case swimmingPool = "SWIMMING POOL"
-    }
-
-    private let headerImageView = UIImageView()
-    private let headerNameLabel = UILabel()
-    private let headerOpenLabel = UILabel()
-    private let headerView = DropdownHeaderView(frame: .zero, arrowImage: nil, arrowImageTrailingOffset: 0)
-
+    private let headerView = FacilitiesDropdownHeaderView(frame: .zero)
     private var collectionView: UICollectionView!
-    private var collectionViewHeight: CGFloat = 0
     private var dropdownView: DropdownView!
     private var facility: Facility!
-    private var height: CGFloat = 0
+    static let headerViewHeight: CGFloat = 52
+    static let collectionViewSpacing: CGFloat = 16
+    private var facilitiesIndex: Int!
+    private var headerViewTapped: ((Int, [Int: Int]) -> ())?
+    private let separatorView = UIView()
+    private let separatorHeight: CGFloat = 1
+    private let separatorSideOffset: CGFloat = 24
+    static let dropdownViewBottomOffset: CGFloat = -12
```

Comment:
Not too familiar with Uplift style but I think it's a bit strange to be mixing static and non-static variables like this. What's the reason for doing so?